### PR TITLE
fix wrong events working for element created in top window and added to an iframe

### PR DESCRIPTION
**Pull request by miherlosev**

### CODE REVIEW

**Comment by miherlosev** [comment]
FPR
@churkin @AlexanderMoskovkin @helen-dikareva

File: test/client/fixtures/sandbox/event/internal-listeners-test.js [hunk 1]
```diff
@@ -611,3 +617,119 @@ if (browserUtils.isIE && browserUtils.version >= 10) {
...
+        var dispatchedEventFlag = 'hammerhead|event-sandbox-dispatch-event-flag';
+        var counter             = 0;
+
+        testInput.addEventListener('focus', function () {
+            iframe.parentNode.removeChild(iframe);
+            start();
+        });
+
+        testInput.value = 'test';
+        iframe.id = 'test_unique_id_qrsj12vbz';
```
Comment (AlexanderMoskovkin) [inline test/client/fixtures/sandbox/event/internal-listeners-test.js]: imho it's better to add some empty lines between blocks of code: here, [here](https://github.com/DevExpress/testcafe-hammerhead/pull/531/files#diff-b522bdf9c15ce13761bbb3a23dc6a2c9R714), [here](https://github.com/DevExpress/testcafe-hammerhead/pull/531/files#diff-b522bdf9c15ce13761bbb3a23dc6a2c9R724) etc.. It'll be more readable

**Comment by miherlosev** [comment]
FPR

**Comment by helen-dikareva** [comment]
let @VasilyStrelyaev check

**Comment by miherlosev** [comment]
comments and test names was checked by @MargaritaLoseva

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by helen-dikareva** [comment]
Ok then. lgtm

**Comment by miherlosev** [comment]
ping @churkin

**Comment by churkin** [comment]
lgtm

**Pull request merged**